src/gamma_inc.jl

```diff
@@ -0,0 +1,583 @@
+# Published in Journal: ACM Transactions on Mathematical Software (TOMS)
+# Volume 12 Issue 4, Dec. 1986 Pages 377-393
+# doi>10.1145/22721.23109
+
+"""
+    gamma_p(a,x,IND)
+    
+    DLMF: https://dlmf.nist.gov/8.2#E4 , https://dlmf.nist.gov/8.2#E5
+    Wiki: https://en.wikipedia.org/wiki/Incomplete_gamma_function
+    IND --> Accuracy desired ; IND=0 means 14 significant digits accuracy , IND=1 means 6 significant digit and IND=2 means only 3 digit accuracy suffices.
+    gamma_p(a,x) or P(a,x) is the Incomplete gamma function ratio given by : ``1/\\Gamma (a) \\int_{0}^{x} e^{-t}t^{a-1} dt``
+"""
+function gamma_p(a::Float64,x::Float64,ind::Integer)
+    iop = ind + 1
+    acc = acc0[iop]
+    if a<0.0 || x<0.0
+        throw(DomainError((a,x,ind,"`a` and `x` must be greater than 0 ---- Domain : (0,inf)")))
+    elseif a==0.0 && x==0.0
+        throw(DomainError((a,x,ind,"`a` and `x` must be greater than 0 ---- Domain : (0,inf)")))
+    elseif a*x==0.0 
+        if x<=a
+            return 0.0
+        else
+            return 1.0
+        end               
+    end
+    
+    if a >= 1.0
+        if a >= big1[iop]
+            l = x/a
+        if l == 0.0
+            return 0.0
+        end
+        s = 1.0 - l
+        z = -logmxp1(l)
+        if z >= 700.0/a
+            return 0.0
+        end
+        y = a*z
+        rta = sqrt(a)
+        if abs(s) <= e0[iop]/rta
+            z = sqrt(z + z) 
+            if l < 1.0 
+                z=-z
+            return gamma_p_temme_1(a, x, z, ind)
+            end
+        end
+
+        if abs(s) <= 0.4
+            if abs(s) <= 2.0*eps() && a*eps()*eps() > 3.28e-3
+                throw(DomainError((a,x,ind,"P(a,x) or Q(a,x) is computationally indeterminant in this case.")))
+            end
+            c = exp(-y)
+            w = 0.5*erfcx(sqrt(y))
+            u = 1.0/a
+            z = sqrt(z + z) 
+            if l < 1.0 
+            z=-z
+            end
+            if iop == 1
+            return gamma_p_minimax(a,x,z)
+            elseif iop == 2
+            return gamma_p_temme(a,x,z)
+            else
+            t = @horner(z , d00 , d0[1] , d0[2] , d0[3])
+            return gamma_p_temme_1(a, x, z, ind)
+            end
+        end
+        elseif a > x || x >= x0[iop] || isinteger(2*a)  
+            r = rgammax(a,x)
+            if r == 0.0
+                if x <= a
+                    return 0.0
+                else
+                    return 1.0
+                end
+            end 
+            if x <= max(a,alog10)
+                return gamma_p_taylor(a, x, ind)
+            elseif x < x0[iop]
+                return gamma_p_cf(a, x, ind)
+            else
+                return gamma_p_asym(a, x, ind)
+            end
+       #else
+       #    return gamma_p_fsum(a,x)
+            
+        end
+    elseif a == 0.5
+        if x >= 0.25
+            return 1.0 - erfc(sqrt(x))
+        end
```

Comment:
I think this block is unnecessary, since `erf(y) == 1-erfc(y)` (my guess is that `erf` was typically defined for a limited domain, but we don't have that problem).